OMCompiler/SimulationRuntime/c/simulation/solver/sundials_util.c

```diff
@@ -78,6 +78,36 @@ void setJacElementSundialsSparse(int row, int column, int nth, double value, voi
   SM_DATA_S(A)[nth] = value;
 }
 
+/**
+ * @brief             Scaling of a sparse matrix column-wise by a vector
+ *
+ * @param A           Sparse matrix in CSC.
+ * @param vScale      Vector for scaling
+ * @return *SUNMatrix Returns the scaled sparse matrix
```

Comment:
This is not a pointer

```suggestion
 * @return SUNMatrix  Returns the scaled sparse matrix
```